paper.md

```diff
@@ -207,10 +207,29 @@ It can either be created by users or provided by system administrators.
 
 # Case Study {#sec:study}
 
-### **Single-Node local workflow execution**
+In this section, we present three case studies of how the Popper workflow engine allows reproducing and scaling workflows easily in different compute environments.
+<!-- briefly describe each case studies here -->
+
+### **Workflow execution on the local machine**
+
+In this section, we discuss how Popper helps researchers to write and test workflows on their local machines and iterate fast.
+We used an Apple Macbook Pro with a 64-bit Intel i5 processor and 8Gb RAM for this casestudy.
+An Image classification workflow was built and run on the MNIST dataset [@deng2012mnist] using the Docker container engine.
+On single node machines, Popper leaves the job of scheduling the containerized steps to the host machines OS.
+We ran the workflow with an overfitting patience of 5 on the laptop's CPU.
+It took around 4035 seconds for the workflow to reach an accuracy of 99.19% over 12 epochs, averaged over 5 test runs.
```

Comment:
* we can add a table to show this and the kubernetes data
  * why did we run only 5? we can add variability over this five executions. Ideally, the stopping condition for iterations is the confidence interval for the variability (e.g. standard deviation).

Reply:
@ivotron  Can you please explain a bit how can i introduce variability, like with some example may be ?

Reply:
I think you can read this http://htor.inf.ethz.ch/publications/img/hoefler-scientific-benchmarking.pdf and then come up with a design and we can discuss it. In general, it means  running some tests, using python tools to obtain non-parametric confidence intervals. Similar to the other project we were working on (confirm)